resources/web/wwi/Parser.js

```diff
@@ -169,7 +170,8 @@ export default class Parser {
             if (typeof parentNode.geometry !== 'undefined')
               parentNode.geometry.delete();
             parentNode.geometry = result;
-          } else if (parentNode instanceof WbSolid || parentNode instanceof WbTransform || parentNode instanceof WbGroup) { // Bounding object
+          // Bounding object
+          } else if (parentNode instanceof WbSolid || parentNode instanceof WbTransform || parentNode instanceof WbGroup) {
```

Comment:
I would rather move the comment inside the  corresponding block
```suggestion
          } else if (parentNode instanceof WbSolid || parentNode instanceof WbTransform || parentNode instanceof WbGroup) {
            // Bounding object
```